alchemist/alchemist-loading/src/test/resources/testgps.yml

```diff
@@ -1,20 +1,20 @@
-environment:
-  type: OSMEnvironment
-  parameters: ["/vcm.pbf", false]
-
-incarnation: sapere
-
-pools:
-  - pool: &move
-    - time-distribution: 0.1
-      type: Event
-      actions:
-        - type: ReproduceGPSTrace
-          parameters: ["/gpsTrace", true, "AlignToSimulationTime"]
-
-displacements:
-  - in:
-      type: FromGPSTrace
-      parameters: [7, "/gpsTrace", true, "AlignToSimulationTime"]
-    programs:
+environment:
+  type: OSMEnvironment
+  parameters: ["vcm.pbf", false]
+
+incarnation: sapere
+
+pools:
+  - pool: &move
+    - time-distribution: 0.1
+      type: Event
+      actions:
+        - type: ReproduceGPSTrace
+          parameters: ["gpsTrace", true, "AlignToSimulationTime"]
+
+displacements:
+  - in:
+      type: FromGPSTrace
+      parameters: [7, "gpsTrace", true, "AlignToSimulationTime"]
+    programs:
```

Comment:
The fact that this file is entirely rewritten makes me think it got the wrong line endings...

Reply:
The original file has Windows line ending and I have configured my git to automatic replace CRLF with LF. Do i have to maintain CRLF?

Reply:
Nope, that's actually great, you are fixing a wrongly formatted file. Consider this solved.